Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8-jdk-alpine
+EXPOSE 8080
+VOLUME /tmp
+ARG JAR_FILE
+COPY ${JAR_FILE:-target/combalph-1.0.0.jar} app.jar
+ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```

Comment:
c4	2f	5a	98	59	42	66	4c	0c	a9	f2	6a	9f	4e	27

Reply:
Thank you for your comment, I'll take it into account following commits